packages/backend/migrations/20241004010448_document_refs.up.sql

```diff
@@ -11,5 +11,6 @@ CREATE TABLE refs (
     created TIMESTAMPTZ NOT NULL
 );
 
-ALTER TABLE snapshots
-    ADD FOREIGN KEY (for_ref) REFERENCES refs (id) DEFERRABLE INITIALLY DEFERRED;
+
+
+    
```

Comment:
Why are we modifying a migration? That should be avoided whenever possible, the migration files are hashed and stored in `_sqlx_migrations.checksum`, and migrations will fail to apply if the checksums don't match.

It is possible to do this, but it's a real pain.